frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-overview/experiment-overview.component.scss

```diff
@@ -49,7 +49,7 @@
     margin-top: -3px;
     margin-left: -12px;
   }
-
+  /* TODO(mdc-migration): The following rule targets internal classes of chips that may no longer apply for the MDC version. */
```

Comment:
@zackcl do you think this rule is of any importance now?

Reply:
Everything looked fine without those rule, so I removed it.